userguide/tasks.rst

```diff
@@ -934,11 +934,10 @@ that dataset is also created.
 Created replication tasks are displayed in
 :menuselection:`Tasks --> Replication Tasks`.
 The task settings that are shown by default can be adjusted by opening
-the :guilabel:`COLUMNS` drop-down. To see more details about the last
-time the replication task ran, click the entry under the
-:guilabel:`State` column. Tasks can also be expanded by clicking
-|ui-chevron-right| for that task. Expanded tasks show all replication
-settings and have |ui-task-run|, |ui-edit|, and |ui-delete| buttons.
+the :guilabel:`COLUMNS` drop-down. Tasks can also be expanded by
+clicking |ui-chevron-right| for that task. Expanded tasks show all
+replication settings and have |ui-task-run|, |ui-edit|, and |ui-delete|
```

Comment:
We need to work toward mentioning capabilities generally rather than using lists of specific buttons or menu entries.  In this case, I'm not even sure this sentence is needed, particularly if a screenshot shows an expanded task.  If we don't have that screenshot, we should.